### DISCUSSION TITLE
New possibilities with the new synth engine

### DISCUSSION THREAD

**Discussion by maxtribe** [bangcorrupt/hacktribe#94] [General]
Hello folks

I have another question about the new Synth Engine in the Hacktribe Firmware. I wanted to know what new possibilities you have with this one, because I'm currently looking for a way to exhaust it. Or what do you use the new synth engine for. I mainly do techno and want to make my patterns as interesting as possible. I still have the Microfreak and then the TD3, but the main device and midi clock is the Electribe. Something along those lines only in techno

Thank you for the information

Warm greetings

PS: My goal is to get a pattern like the one in the following video:

https://www.youtube.com/watch?v=bo5ZBcEGQN8&t=108s

**Comment by bangcorrupt** [comment]
Hi, how you use it is up to you.  I would love to hear more of how people are making use of new features.  Make some beats and post a link.  It's a while since I made any music.

**Reply by Maxklui** [reply]
@bangcorrupt thank you for the answer. But I have one question. How can I install the new templates for example the new Groove types or und in General and
how can I import/export IFX? 
Thank you.

**Reply by bangcorrupt** [reply]
All of the grooves and FX that currently exist are already in hacktribe.  Importing, exporting and editing them currently requires python programming.  The `main()` function of [e2_groove_template.py](https://github.com/bangcorrupt/e2-scripts/blob/2e03bc7fdac69c2fce1a07a159b7a74dda51989d/e2_groove_template.py) demonstrates how to get, edit and set a groove template.  Storing these to flash will be added later.   [e2sysex.py](https://github.com/bangcorrupt/e2-scripts/blob/2e03bc7fdac69c2fce1a07a159b7a74dda51989d/e2sysex.py) includes functions to `get_ifx` and `set_ifx`; there is not yet an editor for FX.  In the near future, the [GUI](https://github.com/bangcorrupt/hacktribe/discussions/97) will allow editing grooves and FX.

**Reply by ptowns** [reply]
Fellow electribe 2 fan and techno head here! If your on IG add me @fake.work. 
Have not posted much lately, had intended to do a YouTube series basically showing my tips tricks and appreciation for the electribe to from a techno perspective. If I ever get around to doing this I will definitely share links here.

**Reply by maxtribe** [reply]
@ptowns Yes I will add you on Instagram.
